Disable optional dependencies (prevents the package from accidentally
finding them if they are not wanted). This should be turned into a proper
robotpkg option.

--- CMakeLists.txt~	2013-01-20 20:47:10.000000000 +0100
+++ CMakeLists.txt	2013-04-19 11:10:23.432672436 +0200
@@ -44,10 +44,10 @@
 ADD_REQUIRED_DEPENDENCY("sot-core >= 1.0")
 
 # Search for closed-source packages.
-ADD_OPTIONAL_DEPENDENCY("hrp2-dynamics >= 1.3.0")
-ADD_OPTIONAL_DEPENDENCY("hrp2-10-optimized >= 1.0")
+#ADD_OPTIONAL_DEPENDENCY("hrp2-dynamics >= 1.3.0")
+#ADD_OPTIONAL_DEPENDENCY("hrp2-10-optimized >= 1.0")
 
-ADD_OPTIONAL_DEPENDENCY("hrp2-10 >= 1.0.0")
+#ADD_OPTIONAL_DEPENDENCY("hrp2-10 >= 1.0.0")
 
 ADD_SUBDIRECTORY(src)
 ADD_SUBDIRECTORY(include)
